### ISSUE TITLE
More flexible masking option

### ISSUE THREAD

**Issue by dr-guangtou** [alexieleauthaud/DeltaSigmaPipelineHSC#53]
Currently, the masking method is based on a string that describes a series of operations (>, <).  

However, there are cases when we need to use more complex and flexible sample selection method (e.g. Top N for certain properties, or in an elliptical region on the M100-M10 plane).  

The easiest way to achieve this is to allow the user to input an additional numpy array as mask.  I propose to: 

1.  Allow a new configuration parameter `external_mask` carries the `.npy` file for the mask array. 
2. When this parameter exists, ignore the mask string, and use this mask directly.

**Comment by dr-guangtou** [comment]
The best solution for this, I think, is to have the `runComputeDS` function available without the need to run a script and prepare a configuration file.  

To have a `stack_ds.py` module can solve this.  For example: 

```
from stack_ds import stack_delta_sigma

# Get the data
dsigma_dir = os.path.join(envir['s16a'], 'um2/s16a_massive_wl')

lens_data = np.load(
    os.path.join(dsigma_dir, 's16a_wide2_massive_fsps1_imgsub_use_short_prep_new.npy'))
lens_ds = np.load(
    os.path.join(dsigma_dir, 's16a_wide2_massive_fsps1_imgsub_use_short_prep_ds_strict.npz'))
rand_ds = np.load(
    os.path.join(dsigma_dir, 's16a_random_100000_prep_ds_strict.npz'))

# Get the mask 
lens_mask = lens_data['logm_100'] > 11.9

# Get the stacked delta sigma result 

dsigma_out = stack_delta_sigma(lens_data, lens_ds, 
                                                       rand_ds=rand_ds, lens_mask=lens_mask)
```

**Comment by dr-guangtou** [comment]
This issue is addressed in recent PR #56, will close.
